dhis-2/dhis-services/dhis-service-audit-consumer/pom.xml

```diff
@@ -0,0 +1,62 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  
+  <parent>
+    <groupId>org.hisp.dhis</groupId>
+    <artifactId>dhis-services</artifactId>
+    <version>2.39-SNAPSHOT</version>
+  </parent>
+  
+  <artifactId>dhis-service-audit-consumer</artifactId>
```

Comment:
why was it deleted before?

Reply:
Error 😬 
It is not directly used anywhere so I thought it was just floating there but this is not the case.
@mortenoh knows more about it and how it is used

Reply:
Yes, this was an error and needs to be merged back immediately. This disabled all audits in DHIS2.